pele_platform/Frag/libraries.py

```diff
@@ -1,71 +1,119 @@
 import glob
 import os
 import subprocess
+import shutil
 
+from rdkit import Chem
 from pele_platform.constants import constants as cs
-import pele_platform.Utilities.Helpers.helpers as hp
 
 OUTPUT = "input.conf"
 
-def growing_sites(fragment, user_bond):
+def get_symmetry_groups(mol):
     """
-    Retrieves all possible growing sites (hydrogens) on the fragment. Takes PDB fragment file as input.
-    Output - list of strings represeting sites, e.g. "benzene.pdb C6-H6 C1-H2"
+    Computes the symmetry class for each atom and returns a list with the idx of non-symmetric atoms.
+
+    Parameters
+    ----------
+    mol: Rdkit molecule object.
```

Comment:
Sorry to bother you again with docstrings but it is important to fulfil the right format from the first beginning. Check out the format [here](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard). Specially the section were it explains the rules to follow for Parameters and Returns:
```
Parameters
----------
x : type
    Description of parameter `x`.
```
Please, note the whitespaces that are in both sides of the colon. Also, look how they specify the type next to the variable name and they describe it below.

We need to satisfy these standards if we want the docstrings to be parsed correctly later by Sphinx.